app/src/main/java/com/heapixLearn/discovery/domain/contact/DumbDBContact.java

```diff
@@ -0,0 +1,45 @@
+package com.heapixLearn.discovery.domain.contact;
+
+import com.heapixLearn.discovery.logic.contact.contacts_domain_interfaces.DBStoreContact;
+
+public class DumbDBContact implements DBStoreContact {
```

Comment:
@dianaprimshits unused class, should be removed

Reply:
@YodaMike Fixed